README.md

```diff
@@ -51,7 +51,8 @@ services:
 | Variable | Description | Required | Default |
 |----------|-------------|---------|--------|
 | `TZ` | Your timezone | No | UTC |
-| `SERIAL` | The serial number of your RTL-SDR dongle | Yes | Blank |
+| `SERIAL` | The serial number of your RTL-SDR dongle. Exactly one of `SERIAL` and `SOAPYSDR` shoudl be set. | No | Blank |
```

Comment:
The wording is slightly confusing. I'd suggest "Exactly one of `SERIAL` or `SOAPYSDR` should be set."

Reply:
Also, do we check anywhere to ensure only one of those two is set, and that at least one is set?

Reply:
check is done for at least one, but not more than one (SERIAL takes precedence). Might be worth considering translating SERIAL into SOAPYSDR, if present, and removing the native rtl-sdr backend for simplicity's sake? 

Fixed the wording (and typo)